src/lib/server/generateFromDefaultEndpoint.ts

```diff
@@ -87,7 +87,12 @@ export async function generateFromDefaultEndpoint(
 	// Close the reader when done
 	reader.releaseLock();
 
-	const results = await JSON.parse(result);
+	let results;                                                                                                                                                                                                                                                                                            
+	if (result.startsWith("data:")) {                                                                                                                                                                                                                                                                       
+		results = [JSON.parse(result.split("data:").pop())]                                                                                                                                                                                                                                                                 
```

Comment:
A `await JSON.parse` may be needed here ?

Reply:
you mean add `await` in front of `JSON.parse`? I don't think it's needed JSON.parse is not async, is it?

Reply:
Yes you're right, so the original results = await JSON.parse(result) might be useless